fcd/python/bindings.h

```diff
@@ -23,7 +23,7 @@
 #define fcd__python_bindings_h
 
 #include <memory>
-#include <Python/Python.h>
+#include <Python.h>
```

Comment:
No, because Python is linked as a framework on Mac OS X. The `Python/` part does not refer to a physical directory.